client/X11/xf_keyboard.c

```diff
@@ -74,7 +74,8 @@ static BOOL xf_keyboard_action_script_init(xfContext* xfc)
 
 	while (fgets(buffer, sizeof(buffer), keyScript) != NULL)
 	{
-		strtok(buffer, "\n");
+		char* context = NULL;
```

Comment:
Shouldn’t this be outside of the loop?

Reply:
no, `strtok` is never called with a `NULL` argument, so each tokenize is a new context